Configuration/DataProcessing/python/RecoTLR.py

```diff
@@ -60,9 +60,14 @@ def customiseDataRun2Common(process):
 def customiseDataRun2Common_25ns(process):
     process = customiseDataRun2Common(process)
 
+    import RecoLocalCalo.HcalRecAlgos.RemoveAddSevLevel as HcalRemoveAddSevLevel
+    HcalRemoveAddSevLevel.AddFlag(hcalRecAlgos,"HFDigiTime",8)
```

Comment:
Hi Slava,

runTheMatrix.py -l 134.802

yielded:
Step2-FAILED Step3-NOTRUN  - time date Fri Sep 25 14:36:26 2015-date Fri 
Sep 25 14:32:12 2015; exit: 0 0 256 0

1 1 0 0 tests passed, 0 0 1 0 failed

Step 2 failed with "tons" of 
"Found an invalid decl for ..."

I think this is known issue irrelevant to PR? 
Salavat



On Fri, 25 Sep 2015, Slava Krutelyov wrote:

> 
> In Configuration/DataProcessing/python/RecoTLR.py:
> 
> > @@ -60,9 +60,14 @@ def customiseDataRun2Common(process):
> >  def customiseDataRun2Common_25ns(process):
> >      process = customiseDataRun2Common(process)
> > 
> > +    import RecoLocalCalo.HcalRecAlgos.RemoveAddSevLevel as HcalRemoveAddSevLevel
> > +    HcalRemoveAddSevLevel.AddFlag(hcalRecAlgos,"HFDigiTime",8)
> Hi Salavat, please use runTheMatrix.py -l 134.802 this is one of the 2015C relvals
> … On 9/25/15 6:50 AM, Salavat Abdullin wrote: In
> Configuration/DataProcessing/python/RecoTLR.py
> <https://github.com/cms-sw/cmssw/pull/11481#discussion_r40421749>: > @@ -60,9 +60,14 @@
> def customiseDataRun2Common(process): > def customiseDataRun2Common_25ns(process): >
> process = customiseDataRun2Common(process) > > + import
> RecoLocalCalo.HcalRecAlgos.RemoveAddSevLevel as HcalRemoveAddSevLevel > +
> HcalRemoveAddSevLevel.AddFlag(hcalRecAlgos,"HFDigiTime",8) Slava, could you just give
> your option (example)? I have only complied and run 'runTheMatrix --useInput all &'
> which obviously don't act on 25ns data. — Reply to this email directly or view it on
> GitHub <https://github.com/cms-sw/cmssw/pull/11481/files#r40421749>.
> 
> —
> Reply to this email directly or view it on
> GitHub.[AEx02vejNIswg-m6_lEKb6u8Ydq5ciH7ks5o1TO0gaJpZM4GDoQK.gif]
> 
> 
>